src/lms-harmonizer/README.md

```diff
@@ -0,0 +1,34 @@
+# lms-harmonizer
+
+
+## Getting Started
+
+## Running the Tool
+
+## Developer Notes
+
+### Dev Operations
+
+1. Style check: `poetry run flake8`
+1. Static typing check: `poetry run mypy .`
+1. Run unit tests: `poetry run pytest`
+1. Run unit tests with code coverage: `poetry run coverage run -m pytest`
+1. View code coverage: `poetry run coverage report`
+
+_Also see
+[build.py](https://github.com/Ed-Fi-Exchange-OSS/LMS-Toolkit/blob/main/docs/build.md)_ for
+use of the build script.
+
+## Legal Information
+
+Copyright (c) 2021 Ed-Fi Alliance, LLC and contributors.
+
+Licensed under the [Apache License, Version 2.0](https://github.com/Ed-Fi-Exchange-OSS/LMS-Toolkit/blob/main/LICENSE) (the "License").
+
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+
+See [NOTICES](https://github.com/Ed-Fi-Exchange-OSS/LMS-Toolkit/blob/main/NOTICES.md) for
+additional copyright and license notifications.
```

Comment:
Do you have the editor config plugin installed in VS Code? If not, please install it. 

Our `.editorconfig` file instructs the plugin to automatically include an extra blank line at the end of the file, which makes the file more readable on Linux.

Reply:
this is interesting, locally I was able to see the blank line but here it didn't show... I added a second line locally and now it shows one here.

Reply:
Oh, maybe GitHub was hiding it. Will be interesting what I find in a few minutes when I download locally.

Reply:
My bad, this is GitHub's doing.